src/parameters/add_parameters.jl

```diff
@@ -41,6 +64,9 @@ function add_parameters!(
     U <: Union{Vector{D}, IS.FlattenIteratorWrapper{D}},
     W <: AbstractDeviceFormulation,
 } where {D <: PSY.HybridSystem}
+    if get_rebuild_model(get_settings(container)) && has_container_key(container, T, D)
```

Comment:
Is there documentation that explains this to model developers?

Reply:
And is there a way to automate this so that it doesn't have to live everywhere?

Reply:
not yet :\